android/src/main/kotlin/com/sap/gigya_flutter_plugin/GigyaFlutterPlugin.kt

```diff
@@ -15,25 +13,23 @@ import io.flutter.plugin.common.MethodChannel.Result
 /** GigyaFlutterPlugin */
 class GigyaFlutterPlugin : FlutterPlugin, MethodCallHandler {
 
-    lateinit var sdk: GigyaSDKWrapper<*>
+    private lateinit var sdk: GigyaSDKWrapper<*>
 
     /// Main communication method channel.
     private lateinit var channel: MethodChannel
-    
-    private var _messenger: BinaryMessenger? = null
 
-    fun <T : GigyaAccount> registerWith(application: Application, accountObj: Class<T>, messenger: BinaryMessenger) {
```

Comment:
This method is required in order to register the custom account schema (if available).